frontend/packages/ceph-storage-plugin/src/utils/install.ts

```diff
@@ -113,12 +113,31 @@ export const countNodesPerZone = (nodes: NodeKind[]) =>
 export const nodesWithoutTaints = (nodes: NodeKind[]) =>
   nodes.filter((node: NodeKind) => hasOCSTaint(node) || hasNoTaints(node));
 
+export const isNormalSC = (
+  sc: StorageClassResourceKind,
+  pvData: K8sResourceKind[],
+  nodesData: NodeKind[],
+  scName?: string,
+): boolean => {
+  const pvs: K8sResourceKind[] = getSCAvailablePVs(pvData, scName || getName(sc));
+  const scNodeNames = getAssociatedNodes(pvs);
+  const tableData: NodeKind[] = nodesData.filter(
+    (node: NodeKind) =>
+      scNodeNames.includes(getName(node)) ||
+      scNodeNames.includes(node.metadata.labels?.['kubernetes.io/hostname']),
+  );
+  const uniqSelectedNodesZones: Set<string> = new Set(tableData.map(getZone));
+  if (_.compact([...uniqSelectedNodesZones]).length < 3) return false;
+  return true;
+};
+
 export const isArbiterSC = (
   sc: StorageClassResourceKind,
   pvData: K8sResourceKind[],
   nodesData: NodeKind[],
+  scName?: string,
```

Comment:
Its better to update the rest of places to get `scName` instead of whole storageClass data.
I can see only name is getting used in this function